Makefile.include

```diff
@@ -487,7 +488,7 @@ endif
 ifeq ($(RIOT_CI_BUILD),1)
     RIOT_VERSION ?= buildtest
     # set a dummy version number
-    RIOT_VERSION_CODE ?= RIOT_VERSION_NUM\(2042,5,23,0\)
+    RIOT_VERSION_CODE ?= ${RIOT_VERSION_DUMMY_CODE}
```

Comment:
I don't know what exactly, but I think there is a syntactic difference between `${var}` and `$(var)` in Make. In other places we use `$(var)` (see `$(RIOT_CI_BUILD)` above e.g.) so even if there is no difference, this should be 

```suggestion
    RIOT_VERSION_CODE ?= $(RIOT_VERSION_DUMMY_CODE)
```